include/openmc/output.h

```diff
@@ -5,47 +5,60 @@
 #define OPENMC_OUTPUT_H
 
 #include <string>
+#include <utility> // for pair
 
 #include "openmc/particle.h"
 
 namespace openmc {
 
-//==============================================================================
+//! \brief Display the main title banner as well as information about the
+//! program developers, version, and date/time which the problem was run.
+void title();
+
 //! Display a header block.
-//!
+//
 //! \param msg The main text of the header
 //! \param level The lowest verbosity level at which this header is printed
-//==============================================================================
-
 void header(const char* msg, int level);
 
-//==============================================================================
 //! Retrieve a time stamp.
-//!
+//
 //! \return current time stamp (format: "yyyy-mm-dd hh:mm:ss")
-//==============================================================================
-
 std::string time_stamp();
 
-//==============================================================================
 //! Display the attributes of a particle.
-//==============================================================================
-
 extern "C" void print_particle(Particle* p);
 
-//==============================================================================
 //! Display plot information.
-//==============================================================================
-
 void print_plot();
 
-//==============================================================================
 //! Display information regarding cell overlap checking.
-//==============================================================================
-
 void print_overlap_check();
 
-extern "C" void title();
+//! Display information about command line usage of OpenMC
+void print_usage();
+
+//! Display current version and copright/license information
+void print_version();
+
+//! Display header listing what physical values will displayed
+void print_columns();
+
+//! Display information about a generation of neutrons
+void print_generation();
+
+//! \brief Display last batch's tallied value of the neutron multiplication
+//! factor as well as the average value if we're in active batches
+void print_batch_keff();
+
+//! Display time elapsed for various stages of a run
+void print_runtime();
+
+//! Display results for global tallies including k-effective estimators
+void print_results();
+
+//! Calculate the mean and standard deviation for a tally result
+std::pair<double, double> mean_stdev(const double* x, int n);
```

Comment:
`mean_stdev` is never used outside output.cpp, right?  If so, it might be better to just forward declare it directly in the .cpp file.  I feel like .h files are more meant for defining an interface than for forward declaring stuff.  And that way, we wouldn't have the `#include <utility>` pulled into every translation unit that uses `#include "openmc/output.h"`

Reply:
I agree -- I'll remove the declaration from the .h file